src/docs/analytics.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Analytics"
+---
+
+This guide steps you through instrumenting your code with Sentry's analytics infrastructure.
+
+## Big Query
+
+Send events to Big Query and run queries in [Redash](https://redash.getsentry.net/).
+
+Create a new file in `src/sentry/analytics/events` using the key in snake case as the filename.
+
+In this file, you will need to register the event with attributes (for validation).
+```python
+from sentry import analytics
+
+class CodeownersAssignment(analytics.Event):
+    type = "codeowners.assignment" # replace with your key
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_id"),
+        analytics.Attribute("group_id"),
+        # more attributes
+    )
+
+analytics.register(CodeownersAssignment)
+```
+
+Now in the code that you want instrumented, use the `analytics.record` function with the registered key.
+```python 
+from sentry import analytics
+
+analytics.record(
+    "codeowners.assignment", # replace with your key
+    organization_id=project.organization_id,
+    project_id=project.id,
+    group_id=group.id,
+    # more attributes
+)
+```
+
+## Datadog
```

Comment:
I don't think this necessarily needs to be Datadog: https://develop.sentry.dev/services/metrics/.

We support a bunch of metrics engines.

Reply:
@manuzope hmm I didn't know that. I still think its relevant if you want an example of sending events to Datadog. But I think we should link to that page, with some context that metrics.incr can be used with any backend.

Reply:
Yea, I think the section is used. I would just make the header "StatsD" or "Metrics". And avoid using "Datadog".

Reply:
I'd go with metrics.